index.js

```diff
@@ -272,21 +272,22 @@ module.exports = {
       addonNamespace: addonInfo.name.classified,
       blueprintVersion: require('./package.json').version,
       year: date.getFullYear(),
-      yarn: options.yarn,
-      pnpm: options.pnpm,
-      npm: options.npm,
+      packageManager: options.packageManager,
+      yarn: isYarn(options),
```

Comment:
👍

Reply:
For my understanding: [this change in ember-cli](https://github.com/ember-cli/ember-cli/pull/10368/files#diff-3b78b0c13cbc4f5b29b0add1dd55afbb2a218144597e6abfb1c17e0c041f4689) is only not a breaking change for _users_, because things like `--pnpm` are still supported as an alias, but it is a breaking change _for us_, because `--pnpm` is not exposed to us as `options.pnpm` anymore but as `options.packageManager` (as of 5.4). Is that correct?